cmd/server/main.go

```diff
@@ -29,43 +31,57 @@ func main() {
 		log.Fatal(err)
 	}
 	config.Params = cnf
-	wg := sync.WaitGroup{} // Группа для синхронизации
+	wg := new(errgroup.Group)
+	//wg := sync.WaitGroup{} // Группа для синхронизации
 
 	// стартуем приложение
-	if err = runApplication(&wg); err != nil {
+	if err = runApplication(wg); err != nil {
```

Comment:
На мой взгляд не совсем удачное использование `errgroup.Group`, так как не видно, как она вызывается. Мне кажется стоит сделать вызовы видимым, либо использовать `wg.Wait()` там, где вызывается `Go(...)` и адаптировать код для этого.
```suggestion
	wg.Go(func() error { return runApplication() })
```

Reply:
Сделал использование группы только в одном методе